# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rpcbind/hotfix-gcc14.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Mostly for musl.

--- rpcbind-1.2.5/src/pmap_svc.c.vanilla	2024-07-25 12:59:22.220784785 +0200
+++ rpcbind-1.2.5/src/pmap_svc.c	2024-07-25 12:59:35.862784620 +0200
@@ -46,6 +46,7 @@
  * All the portmapper related interface from the portmap side.
  */
 
+#include <string.h>
 #ifdef PORTMAP
 #include <sys/types.h>
 #include <sys/socket.h>
--- rpcbind-1.2.5/src/warmstart.c.vanilla	2024-07-25 12:59:58.090784352 +0200
+++ rpcbind-1.2.5/src/warmstart.c	2024-07-25 13:00:05.293784265 +0200
@@ -32,6 +32,7 @@
  * Copyright (c) 1990 by Sun Microsystems, Inc.
  */
 
+#include <string.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <stdio.h>
